src/front/glsl_new/parser.rs

```diff
@@ -96,6 +133,9 @@ pomelo! {
                     }),
                 },
             );
+            extra.get_context().expressions.append(Expression::GlobalVariable(h))
+        } else {
+            return Err(ErrorKind::NotImplemented(String::from("var")))
```

Comment:
I first tried without the `return`, but that gives compiler error. I think this might be a pomelo quirk, like it only sets return type to `Result` if it sees `return Err`